### DISCUSSION TITLE
Retorno dos meetups Python MG

### DISCUSSION THREAD

**Discussion by daniel-canesso** [apyb/comunidade#105] [Divulgação]
É real: depois de anos sem meetups presenciais da nossa comunidade, faremos um em Belo Horizonte!

![image](https://github.com/apyb/comunidade/assets/38555418/5dc6cec6-c9a9-4e58-b310-280b1ea2d9f5)

Novidades serão divulgadas em breve!

**Comment by ayr-ton** [comment]
@daniel-canesso Onde vai ser?

**Comment by daniel-canesso** [comment]
@ayr-ton será em Belo Horizonte, no edificio da Seculus, dia 28/02/2026.
https://www.meetup.com/pythonmg/events/298629689/
